src/core/Akka/Util/MonotonicClock.cs

```diff
@@ -20,31 +19,33 @@ namespace Akka.Util
     /// </summary>
     internal static class MonotonicClock
     {
-        private static readonly Stopwatch Stopwatch = Stopwatch.StartNew();
-
-        private const int TicksInMillisecond = 10000;
+        private static readonly Stopwatch Stopwatch;
 
+        private const long TicksInMillisecond = TimeSpan.TicksPerMillisecond;
+        private const long TicksInSecond = TimeSpan.TicksPerSecond;
         private const long NanosPerTick = 100;
 
+        private static readonly double TicksFrequency;
+        
+        static MonotonicClock()
+        {
+            TicksFrequency = (double)TicksInSecond / Stopwatch.Frequency;
```

Comment:
Why is this here? Please add a comment in the code to explain, as it's non-obvious why this is needed.

Reply:
`TickFrequency` is a constant scaling value to normalize operating system reported performance counter clock tick to .NET internal definition of a "tick".

`Stopwatch.ElapsedTicks` returns the raw operating system level performance clock ticks, which is not the same definition as .NET `DateTime` or `TimeSpan` definition.

.NET `DateTime` or `TimeSpan` definition of a "tick" is 100 nanosecond, which is true for all windows platforms that support performance counter clock (win8+). This is not true for linux platforms, their definition of a "tick" is 1 nanosecond.